apps/emqx_durable_storage/test/emqx_ds_storage_bitfield_lts_SUITE.erl

```diff
@@ -211,10 +224,7 @@ t_replay(_Config) ->
     ok = emqx_ds_storage_layer:store_batch(?SHARD, Batch1, []),
     %% Create wildcard topics `wildcard/+/suffix/foo' and `wildcard/+/suffix/bar':
     Batch2 = [
-        begin
-            Topic = emqx_topic:join(["wildcard", integer_to_list(I), "suffix", Suffix]),
-            {TS, make_message(TS, Topic, integer_to_binary(TS))}
-        end
+        {TS, make_message(TS, make_topic([wildcard, I, suffix, Suffix]), bin(TS))}
```

Comment:
Same here, TS is probably not needed.

Reply:
Had a different impression actually: sounds important for a replay test to have messages spread both in time and in "topic space" (here, `TS` is not part of a topic).

Anyway, it'd probably be better to tackle the logic of unrelated testcase in a separate PR.